### ISSUE TITLE
BUG - Missing textures 

### ISSUE THREAD

**Issue by nodrog1061** [Seb105/Arma-Briefingtable#8]
Some textures like taxiway textures don't load in and show as missing textures textures

**Comment by Seb105** [comment]
Example location?

**Comment by nodrog1061** [comment]
I can't give you a photo but if you look at the airport on altis you'll be able to see what I mean

**Comment by Seb105** [comment]
Missing textures are invisible, they show "missing textures". I think the taxiway texture just looks bad when scaled to a cube. Not a lot that can be done really.

**Comment by nodrog1061** [comment]
thats fair enough i found an example it's in your video you put  on the workshop
![image](https://user-images.githubusercontent.com/59317189/111051577-015a2d00-844c-11eb-8995-5686ef3e9a70.png)

**Comment by Seb105** [comment]
Yeah that is the texture of the taxiway road (not necessarily the concrete underneath it). By design the code picks up road textures as it works better for towns and stuff. I'll leave this issue open but it's not a priority.
